src/analyzer/protocol/ldap/ldap.spicy

```diff
@@ -267,13 +264,18 @@ public type Message = unit {
     ProtocolOpcode::INTERMEDIATE_RESPONSE   -> INTERMEDIATE_RESPONSE:   NotImplemented(self);
     ProtocolOpcode::MOD_DN_REQUEST          -> MOD_DN_REQUEST:          NotImplemented(self);
     ProtocolOpcode::SEARCH_RESULT_REFERENCE -> SEARCH_RESULT_REFERENCE: NotImplemented(self);
-  } &parse-from=self.applicationBytes if ( self.opcode );
+  } &size=self.protocolOp.len.len;
```

Comment:
NB: Ugly, but this is really due to how the ASN.1 types look. In a more ideal world they would extract more "natural" values. Similar for `self.seqHeader.len.len` below.